docs/section21/Rule_21-2.md

```diff
@@ -0,0 +1,39 @@
+
+# Boiler - Rule 21-2  
+
+**Rule ID:** 21-2  
+**Rule Description:** For purchased HW/steam in the proposed model, the baseline shall have the same number of pumps as proposed
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.1.1 & G3.1.1.3.1 Building System-Specific Modeling Requirements for the Baseline model  
+**Schema Version:** 0.0.25
```

Comment:
Both sections  G3.1.1.1 & G3.1.1.3.1 do not have the requirement 'For purchased HW/steam in the proposed model, the baseline shall have the same number of pumps as proposed'
G3.1.1.3.4 On-Site Distribution Pumps- says 'All on-site distribution pumps shall be modeled in both the proposed design and base building design'

Do we need to update both the ruleset reference and the rule description?

Reply:
SG to confirm that Section 22 has a corresponding rule.

Reply:
updated to G3.1.1.3.4

Reply:
SG confirmed that Section 22 has a rule which addresses this ffor PCHW. 
SG has also updates the rules tab for Section 21 rule test spreadsheet with correct ruleset reference.